src/ScottPlot5/ScottPlot5/Plottables/Pie.cs

```diff
@@ -105,6 +106,28 @@ public override void Render(RenderPack rp)
                 slice.LabelStyle.Render(rp.Canvas, px, paint);
             }
 
+            var slicePercentFormat = SlicePercentFormat;
+            if (slicePercentFormat != null &&
+#if NET5_0_OR_GREATER
+                slicePercentFormat.StartsWith('P') // Just follow the code quality rule CA1865
+#else
+                slicePercentFormat.StartsWith("P", StringComparison.Ordinal)
```

Comment:
maybe this can be OrdinalIgnoreCase?

Reply:
No. See [https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#percent-format-specifier-p](https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-numeric-format-strings#percent-format-specifier-p).